subprojects/docs/src/docs/userguide/publishingMaven.adoc

```diff
@@ -175,13 +175,32 @@ If a project defines multiple publications then Gradle will publish each of thes
 Publications are published to repositories. The repositories to publish to are defined by the api:org.gradle.api.publish.PublishingExtension#getRepositories()[] container.
 
 ++++
-<sample dir="maven-publish/quickstart" id="publishing_maven:repositories" title="Declaring repositories to publish to">
+<sample dir="maven-publish/quickstart" id="publishing_maven:example:repositories" title="Declaring repositories to publish to">
     <sourcefile file="build.gradle" snippet="repositories"/>
 </sample>
 ++++
 
 The DSL used to declare repositories for publication is the same DSL that is used to declare repositories to consume dependencies from, api:org.gradle.api.artifacts.dsl.RepositoryHandler[]. However, in the context of Maven publication only api:org.gradle.api.artifacts.repositories.MavenArtifactRepository[] repositories can be used for publication.
 
+[[publishing_maven:snapshot_and_release_repositories]]
+==== Snapshot and release repositories
+
+It is a common practice to publish snapshots and releases to different Maven repositories. A simple way to accomplish this is to configure the repository URL based on the project version.
+
+++++
+<sample dir="maven-publish/javaProject" id="publishing_maven:example:repo-url-from-variable" title="Configuring repository URL based on project version">
+    <sourcefile file="build.gradle" snippet="repo-url-from-variable"/>
+</sample>
+++++
+
+Similarly, you can use a <<build_environment, project or system property>> to decide for one repository or another. The following example uses the release repository if the project property `release` is set, e.g. by running `gradle -Prelease publish`.
```

Comment:
to decide which repository to publish to.

or

    to decide on one repository over another

I think it's also important to clarify what the e.g. refers to. For example:

    The following example/sample uses the release repository if the project property `release` is set, such as when a user runs `gradle -Prelease publish`:

This just seems a little clearer to me.